brainiak/matnorm_models/helpers.py

```diff
@@ -0,0 +1,73 @@
+import functools  # https://danijar.com/structuring-your-tensorflow-models/
+import tensorflow as tf
+from numpy.linalg import cholesky
+from scipy.stats import norm
+
```

Comment:
It could go to "brainiak/utils/tensorflow.py", except for the `rmn`, which could stay in this file, renamed "utils.py".

In any case, we cannot merge this code until we hear from @danijar.